living/kabyle.md

```diff
@@ -2,45 +2,60 @@
 
 Additional names:
 - Taqbaylit
-- 
+- Tazwawt
 
 News:
-- 
-- 
+- https://www.aps.dz/tamazight-tal/
+- https://my.radioalgerie.dz/dz-ber
+- https://www.depechedekabylie.com/ddk-tamazight/
 
 Culture / History:
-- 
-- 
+- https://tasghunt-taseklant.com/kab/
+- http://isahliyen.com (in kabyle and french)
+- https://tamurt.info/tq/category/taqbaylit/
+- https://tamsirt.numidya.net/cours (Courses about language)
 
 Government:
-- 
+- https://www.hcamazighite.dz/ta/
 - 
 
 Political Parties:
 - 
 - 
 
+Podcasts:
+- https://tunein.com/podcasts/Technology-Podcasts/Taqbaylit-p2448080/
+
 Other:
 - https://kab.wikipedia.org
-- 
-- 
+- https://www.mozilla.org/kab/
+- https://www.amyag.com (Reference for verb conjugaison)
+- http://www.akufi.org/kab/ (Mostly about Kabyle keyboard layout)
+- http://www.ayamun.com (in french about kabyle and berber book ressources culture and litterature)
```

Comment:
Small typos here and below

Reply:
fixed conjugaison to conjugation, ressources to resources in english and litterature to literature.